app/models/vacancy.rb

```diff
@@ -268,6 +268,16 @@ def delete_documents
     self.documents.each { |document| DocumentDelete.new(document).delete }
   end
 
+  def new_information
+    sections = []
+    sections << 'supporting_documents' if supporting_documents_new
+    sections
+  end
+
+  def supporting_documents_new
```

Comment:
I like the Ruby convention where a method that returns a Boolean ends with a question mark